src/test/java/gov/cms/ab2d/eventlogger/Ab2dEnvironmentTest.java

```diff
@@ -1,13 +1,14 @@
 package gov.cms.ab2d.eventlogger;
 
 
+
+import gov.cms.ab2d.eventclient.config.Ab2dEnvironment;
 import org.junit.jupiter.api.Test;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 public class Ab2dEnvironmentTest {
```

Comment:
Why is there a need to duplicate the test from the client library here?

If none, please delete.